lib/events/mock.go

```diff
@@ -62,7 +63,7 @@ func (d *MockAuditLog) Close() error {
 }
 
 func (d *MockAuditLog) EmitAuditEvent(event Event, fields EventFields) error {
-	return nil
+	return d.MockEmitAuditEvent(event, fields)
```

Comment:
check that `MockEmitAuditEvent` is not nil

Reply:
revised